examples_notebooks/community_contrib/couchbase/GraphRAG_with_Couchbase.ipynb

```diff
@@ -0,0 +1,656 @@
+{
```

Comment:
Local search indication is missing as vector store is only used for local search.

Reply:
Added in the title